### ISSUE TITLE
Details in CSOC page

### ISSUE THREAD

**Issue by AnonySharma** [COPS-IITBHU/cops-site#11]
- About CSOC Section
- Stats
- Timeline
- Contact Info for Mentors for each sections : DEV, CP, ML, CTF

**Comment by king-11** [comment]
@dasari810 @sanyud24 you can work on csoc page components

**Comment by dasari810** [comment]
yes @king-11  the work is in progress .. i @subodhk01  and @arul-ashri  made a basic template and divided the work among us .. i made a PR  #16  regarding my part .. if any suggestion in that .. please let me know
